azure/scope/managedcontrolplane.go

```diff
@@ -448,6 +448,9 @@ func (s *ManagedControlPlaneScope) ManagedClusterSpec(ctx context.Context) azure
 	if s.ControlPlane.Spec.LoadBalancerSKU != nil {
 		managedClusterSpec.LoadBalancerSKU = *s.ControlPlane.Spec.LoadBalancerSKU
 	}
+	if s.ControlPlane.Spec.OutboundType != nil {
+		managedClusterSpec.OutboundType = *s.ControlPlane.Spec.OutboundType
+	}
```

Comment:
As simple as this is, it'd be great to add a unit test to cover this.

Reply:
done